seqr/views/utils/dataset_utils.py

```diff
@@ -432,10 +426,11 @@ def _load_rna_seq(model_cls, file_path, user, mapping_file, ignore_extra_samples
         sample: samples_by_id[sample.sample_id] for sample in samples if sample.id not in loaded_sample_ids
     }
 
+    filters = {'filter': ~Q(family__individual__id__in=prev_loaded_individual_ids)} if prev_loaded_individual_ids else {}
     sample_projects = Project.objects.filter(family__individual__sample__in=samples_to_load.keys()).values(
         'guid', 'name', new_sample_ids=ArrayAgg(
             'family__individual__sample__sample_id', distinct=True, ordering='family__individual__sample__sample_id',
-            filter=~Q(family__individual__id__in=prev_loaded_individual_ids)
+            **filters,
```

Comment:
The reason for the change is that it fails when `prev_loaded_individual_ids` is empty.

Reply:
"not in empty set" should just return True for all the queries so I don't see why this change is necessary

Reply:
When `prev_loaded_individual_ids` is empty and `samples_to_load` is non-empty, there will be an error with the message below.
```
"message": "syntax error at or near \")\"\nLINE 1: ...ORDER BY \"seqr_sample\".\"sample_id\") FILTER (WHERE ) AS \"new_...
"user": "test_data_manager@test.com",
"traceback": "Traceback (most recent call last):
  File \"/Users/shifa/opt/anaconda2/envs/mito/lib/python3.7/site-packages/django/db/backends/utils.py\", line 84, in _execute
    return self.cursor.execute(sql, params)\npsycopg2.errors.SyntaxError: syntax error at or near \")\"
LINE 1: ...ORDER BY \"seqr_sample\".\"sample_id\") FILTER (WHERE ) AS \"new_...

The above exception was the direct cause of the following exception: ...
```

Reply:
I've just added a test for `loading new data without existing data`. If we don't change the code, the new test will fail.